code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -1273,11 +1273,11 @@ nobiliumsuppression = INFINITY
 	if(cached_gases[/datum/gas/bz][MOLES] - consumed_amount < 0)
```

Comment:
If location is null this should error too. Also I did some thinking about this line, why take the max of 5, BZ, and proto if we're gonna do this check here?